src/ui/identities/update_identity_payout_address.rs

```diff
@@ -0,0 +1,239 @@
+                                .selectable_value(
+                                    &mut self.selected_payout_address,
+                                    Some(address.clone()),
+                                    address.to_string(),
+                                )
+                                .clicked()
+                            {}
+                        }
+                    });
+            });
+            ui.add_space(20.0);
+            if let Some(selected_address) = &self.selected_payout_address {
+                if let Some(value) = wallet.address_balances.get(&selected_address) {
+                    ui.label(format!("Selected Address has a balance of {} DASH", value));
+                } else {
+                    // TODO: Why sometimes balance is not found?
+                    //ui.label("Balance NOT FOUND DASH".to_string());
+                }
+            }
+        }
+    }
+
+    fn render_selected_wallet_funding_addresses(&mut self, ctx: &Context, ui: &mut Ui) {
+        if let Some(selected_wallet) = &self.selected_wallet {
+            // Acquire a read lock
+            let wallet = selected_wallet.read().unwrap();
+            ui.add_space(20.0);
+            ui.heading("Select a Funding Address:");
+            ui.add_space(5.0);
+            ui.push_id("funding_combo_id", |ui| {
+                ComboBox::from_label("")
+                    .selected_text(
+                        self.selected_funding_address
+                            .as_ref() // Get a reference to the Option<Address>
+                            .map(|address| address.to_string()) // Convert Address to String
+                            .unwrap_or_else(|| "".to_string()), // Use default "" if None
+                    )
+                    .show_ui(ui, |ui| {
+                        for (address, _) in &wallet.known_addresses {
+                            if ui
+                                .selectable_value(
+                                    &mut self.selected_funding_address,
+                                    Some(address.clone()),
+                                    address.to_string(),
+                                )
+                                .clicked()
+                            {}
+                        }
+                    });
+            });
+            ui.add_space(20.0);
+            if let Some(selected_address) = &self.selected_funding_address {
+                if let Some(value) = wallet.address_balances.get(&selected_address) {
+                    ui.label(format!("Selected Address has a balance of {} DASH", value));
+                } else {
+                    // TODO: Why sometimes balance is not found?
+                    //ui.label("Balance NOT FOUND DASH".to_string());
+                }
+            }
+        }
+    }
+}
+
+impl ScreenLike for UpdateIdentityPayoutScreen {
+    fn display_message(&mut self, message: &str, _message_type: MessageType) {
```

Comment:
I suggest using an UpdatePayoutAddressStatus enum for the screen and doing display_message like is done in other screens, such as register_dpns_name_screen.rs